bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Thing.java

```diff
@@ -206,4 +207,10 @@
      */
     void setLocation(@Nullable String location);
 
+    /**
+     * Returns {@code true} if the thing is enabled. Return {@code false} otherwise.
+     * 
+     * @return Returns {@code true} if the thing is enabled. Return {@code false} otherwise.
+     */
+    boolean isEnabled();
```

Comment:
Is it expected this information is also exposed by the REST interface?
Is it expected to have also REST resources to enable / disable a thing?